include/quicly/loss.h

```diff
@@ -44,11 +44,16 @@ typedef struct quicly_loss_conf_t {
      * The default RTT used before an RTT sample is taken.
      */
     uint32_t default_initial_rtt;
+    /**
+     * Number of speculative PTOs at the end of a window. This must not be set to more than 3.
+     */
+    uint32_t num_speculative_ptos;
```

Comment:
Maybe `uint8_t`? Other than that, the PR looks awesome. This is superb!

Reply:
Yeah, good point. Done.